wallet/ergvein-wallet.cabal

```diff
@@ -75,6 +78,7 @@ library
     , data-default
     , directory
     , ergvein-common            >= 1.0      && < 1.1
+    , ergvein-index-api
```

Comment:
Need version

Reply:
done in 7dc9b0c75cae355c7eef2ff1d285723d60d0684c